### PR TITLE
Updated documentation to recommend kubernetes 1.5.3 for vSphere Cloud Provider

### PR DESCRIPTION
**Pull request by divyenpatel**
Updated documentation and `phase2/Kconfig `to recommend kubernetes 1.5.3 for vSphere Cloud Provider

cc: @kerneltime @abrarshivani @BaluDontu @tusharnt @pdhamdhere

### CODE REVIEW

**Comment by abrarshivani** [comment]
I would suggest to add this issue (https://github.com/kubernetes/kubernetes-anywhere/issues/337) as known issue in file phase1/vsphere/README.md.

**Comment by BaluDontu** [comment]
@divyenpatel : Can you please have a look at this today.

File: phase1/vsphere/README.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ To access the dashboard after successful instllation of kubernetes cluster. Ther
 
 * Run ```kubectl proxy``` outside the container spawned by ```make docker-dev```
 ```
-curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.4.8/bin/linux/amd64/kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.5.3/bin/linux/amd64/kubectl
```
Comment (abrarshivani) [inline phase1/vsphere/README.md]: Can you please add comment here for user to download the kubectl version that matches with kubernetes cluster they have launched?

Comment (divyenpatel) [inline phase1/vsphere/README.md]: done.

**Comment by abrarshivani** [comment]
LGTM

**Comment by abrarshivani** [comment]
@divyenpatel Can you please squash the commits?

**Pull request merged**